scripts/cluster-helpers.sh

```diff
@@ -275,3 +278,14 @@ function check_prereq() {
 		fi
 	fi
 }
+
+# create kubernetes secret
+function create_secret() {
+
+	namespace="$1"
+	# create a kube secret each time app is deployed
+	kubectl create secret docker-registry hpodockersecret --docker-server=$REGISTRY --docker-username=$REGISTRY_USERNAME \
+	--docker-password=$REGISTRY_PASSWORD --docker-email=$REGISTRY_EMAIL -n ${namespace}
+	# link the secret to the service account
+	oc secrets link default hpodockersecret --for=pull -n ${namespace}
```

Comment:
It looks like this is used by both minikube and OpenShift. We cannot use the `oc` command for minikube. Please use kubectl for both